ui/pages/Staff/constants.js

```diff
@@ -153,3 +165,10 @@ export const DISCOVERY_SHEET_COLUMNS = [
   { name: 'num_individuals_sequenced', content: '# of Individuals Sequenced' },
   { name: 'analysis_summary', content: 'Analysis Summary', style: { minWidth: '800px' } },
 ]
+
+export const SUCCESS_STORY_COLUMNS = [
+  { name: 'family_id', content: 'Family ID', format: formatIDLink, noFormatExport: true, style: { minWidth: '200px' } },
+  { name: 'success_story_types', content: 'Success Story Types', format: formatSuccessStoryTypes, style: { minWidth: '300px' } },
+  { name: 'success_story', content: 'Success Story', style: { minWidth: '564px' } },
+  { name: 'discovery_tags', content: 'Discovery Tags', format: formatDiscoveryTags, noFormatExport: true, style: { minWidth: '400px' } },
+]
```

Comment:
Can you double check that when you download this it looks good?

Reply:
When downloaded, success story types are in short form. Otherwise, it looks fine to me.

Reply:
By short form do you mean the single letter code? Because if so, could you fix that?